lib/broadway_kinesis/subscribe_to_shard.ex

```diff
@@ -432,5 +432,5 @@ defmodule BroadwayKinesis.SubscribeToShard do
        }
 
   defp starting_position_data({:at_timestamp, %DateTime{} = timestamp}),
-    do: %{"Type" => "AT_TIMESTAMP", "Timestamp" => DateTime.to_iso8601(timestamp)}
+    do: %{"Type" => "AT_TIMESTAMP", "Timestamp" => DateTime.to_unix(timestamp)}
```

Comment:
Even though AWS documentation seems to indicate that iso8601 format _should_ work, in practice this was resulting in API errors. Changing it to a unix timestamp does work.

https://docs.aws.amazon.com/kinesis/latest/APIReference/API_StartingPosition.html

Reply:
Is this because the iso8601 string didn't have ms precision?

Reply:
Confirmed. Adding explicit ms precision (and even making sure it's prefixed with full timezone offset, as in the docs) seemingly makes not difference. Unless I'm misunderstanding.

Example: `"2025-07-10T04:00:00.000-00:00"`
Error:
```
%{"Message" => "STRING_VALUE can not be converted to milliseconds since epoch", "__type" => "SerializationException"}
```

I've also tried just sending the raw `DateTime` into the JSON serializer and that seemingly doesn't work either.